test/e2e/data/e2e_conf.yaml

```diff
@@ -76,6 +80,8 @@ providers:
             new: "--enable-leader-election=false"
           - old: --metrics-addr=127.0.0.1:8080
             new: --metrics-addr=:8080
+          - old: "gcr.io/k8s-staging-cluster-api/kubeadm-control-plane-controller:master"
+            new: "quay.io/kinvolk/kubeadm-control-plane-controller-amd64:dev-v1alpha3"
```

Comment:
Please revert these changes

Reply:
This is part of `TMP` commit, which must be removed before merging, but right now it is required to be able to test whole thing. Should I remove it right now anyway?

Reply:
This will be removed before merging.